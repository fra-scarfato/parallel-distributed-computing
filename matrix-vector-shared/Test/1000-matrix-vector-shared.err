/homes/DMA/PDC/2020/SCRFNC00B/matrix-vector-shared/matrix-vector-shared.c:174:2: warning: no newline at end of file
