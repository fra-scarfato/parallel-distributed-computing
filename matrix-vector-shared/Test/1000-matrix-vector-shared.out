Processors number   Time (average)      Speed-up       Efficiency     
1                   0.01075             1.00000        1.00000        
2                   0.00692             1.55332        0.77666        
3                   0.00637             1.68665        0.56222        
4                   0.00596             1.80369        0.45092        
5                   0.00569             1.88811        0.37762        
6                   0.00570             1.88447        0.31408        
7                   0.00615             1.74724        0.24960        
8                   0.00956             1.12449        0.14056        
